lib/changeling/models/logling.rb

```diff
@@ -48,7 +48,7 @@ def klassify(object)
 
         def records_for(object, length = nil, field = nil)
           filters = [
-            { :klass => Logling.klassify(object) },
+            { :klass => Logling.klassify(object).split('/') },
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.